### ISSUE TITLE
Use IEC binary prefixes instead of SI units

### ISSUE THREAD

**Issue by flx42** [hpc/mpifileutils#345]
I was comparing `dcp` with `dd` for a single file, and was initially puzzled by the performance difference with similar block size.

I then realized that `dcp` prints `GB/s` when it is actually `GiB/s`. When this is accounted for, `dd` and `dcp` indeed show the same performance.

In accordance to [IEEE 1541-2002](https://en.wikipedia.org/wiki/IEEE_1541-2002) and [ISO IEC 80000-13](https://en.wikipedia.org/wiki/Binary_prefix#Adoption_by_IEC,_NIST_and_ISO) could we modify the following units to use binary prefixes?
https://github.com/hpc/mpifileutils/blob/57a3578b68d8df0fa53a7662c33c6c21ce72400f/src/common/mfu_util.c#L310-L322 

This is confusing because other I/O tools use the right unit *and* print both types of units to show that they are aware of the potential ambiguity:
dd:
```
419430400 bytes (419 MB, 400 MiB) copied, 0.0436056 s, 9.6 GB/s
```
See https://github.com/coreutils/coreutils/blob/8d13292a73ecf1f265f77731d3ace29866e3d616/src/dd.c#L588-L591

fio:
```
   READ: bw=506MiB/s (530MB/s), 506MiB/s-506MiB/s (530MB/s-530MB/s), io=2048MiB (2147MB), run=4049-4049msec
```
See https://fio.readthedocs.io/en/latest/fio_doc.html#units

**Comment by adammoody** [comment]
Yes, good point.  We should move to adopt those standardized units.

Since this changes the user interface in places where people are allowed to input units, we'll save that change for the next major release (v0.11).
